### PR TITLE
Commerce 1490 option facet config

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-1490

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ehy @ivicac I've started with this PR

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/option_facets/configuration.jsp [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+CPOptionFacetPortletPreferences cpOptionFacetPortletPreferences = new CPOptionFacetPortletPreferences(java.util.Optional.ofNullable(portletPreferences));
```
Comment (ivicac) [inline modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/option_facets/configuration.jsp:20]: @andrea-ale-sbarra please, either put java.util.Optional in imports or create the Optional value inside the CPOptionFacetPortletPreferences constructor

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Pull request closed without merging**